### DISCUSSION TITLE
The route rolls back when it is first loaded 路由回退

### DISCUSSION THREAD

**Discussion by pocketChao** [vuejs/router#2147] [Help and Questions]
### Vue version

3.3.4

### Link to minimal reproduction

项目代码比较多  最小复现链接暂时无法提供

### Steps to reproduce

当我用vite+vue3搭建一个项目的时候 在我每次启动服务的时候  跳转到某个页面都会闪一下然后回退到当前的页面 ，之后再进去就没有问题了 我很困惑不知道是什么导致了这个情况   
const router = createRouter({
  // history: createWebHashHistory(import.meta.env.BASE_URL),
  history: createWebHashHistory(), 

### What is expected?

希望可以正常跳转

### What is actually happening?

我不知道发生了什么  我想问问有人遇到相同的问题吗  这对我们的项目产生了严重的影响

**Comment by bibibala** [comment]
我也遇到同样的问题,我使用的是qusar vite,插眼👀

**Comment by bibibala** [comment]
https://www.cnblogs.com/LannyChung/p/17082865.html 
你的问题可以参考这个,看看是否有用,说是vite的关系,我使用的quasar vite 我没使用自动导入,也遇到同样的问题,还没有解决办法
